ir.ts

```diff
@@ -34,7 +34,7 @@ export type Expr<A> =
 
 export type Value<A> = 
     { a?: A, tag: "num", value: bigint }
-  | { a?: A, tag: "wasmint", value: number }
+  | { a?: A, tag: "wasmint", value: bigint }
```

Comment:
I'm confused about this. We need _something_ that represents plain numbers in generated code, for example small offsets to fields. If we actually put in a bigint here, it would produce non-well-formed WASM code with a literal that is too large.